### DISCUSSION TITLE
Handling daylight savings time changes

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#8424] [General]
I have found an issue relating to the upcoming change from BST to GMT this weekend but I suspect this may be relevant to other changes too.
I have an application where users book a data and that is stored in ISO format (2022-11-02T00:00:00.000) and when that record was added the timezone was BST (GMT+1). 
Now when I try to filter based on that with date > 2022-11-02T00:00:00.000 AND date < 2022-11-02T23:59:59.000 no results are returned as technically the date on that record is now 11pm the day before so shows in the results for the wrong day.

The form that creates the row has a date picker and the 'Ignore time zones' option is set. Is this the problem? Should this not be set?

**Comment by melohagan** [comment]
Hey @andyburgessmd 

You'll need to set **Ignore timezones** on the Date/Time field of your Data table: https://docs.budibase.com/docs/datetime#ignore-time-zones
